sbol/test/test_identified.py

```diff
@@ -102,5 +102,44 @@ def test_name(self):
         self.assertEqual(cd.name, expected_name)
 
 
+class TestCopy(unittest.TestCase):
+
+    def test_copy_properties(self):
+        i = sbol.Identified('i')
+        i.name = 'foo'
+        i_copy = i.copy()
+        self.assertEqual(i_copy.name, 'foo')
+
+    def test_import_object_into_new_namespace(self):
+        # When copying an object into a new namespace, confirm that it's URI is copied into the new namespace.
+        # Also confirm that any ReferencedObject attributes whose values point to an object in the old namespace
+        # are also copied into the new namespace
+
+        setHomespace('http://examples.org')
+        Config.setOption('sbol_typed_uris', False)
+        comp = ComponentDefinition('cd')
+        seq = Sequence('seq')
```

Comment:
Caution. You're using an outdated copy of test_identified.py. The latest version does not wildcard import the sbol package. Please use `sbol.`.

Reply:
Fixed